From 2c80dcd02031c0d0706386717af775e9066743b4 Mon Sep 17 00:00:00 2001
From: gora5624 <gora5624@gmail.com>
Date: Mon, 22 Jun 2020 15:51:12 +0500
Subject: [PATCH] README add

---
 README | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 README

diff --git a/README b/README
new file mode 100644
index 0000000..d26c8e4
--- /dev/null
+++ b/README
@@ -0,0 +1 @@
+Some
\ No newline at end of file
-- 
2.16.2.windows.1

